README.rst

```diff
@@ -211,75 +211,92 @@ To make ``--follow-tags`` permanent run this command: ::
-          -e TRAVIS_BUILD_ID
-          -e TRAVIS_PULL_REQUEST_BRANCH
-          -e TRAVIS_PULL_REQUEST
-          -e TRAVIS_REPO_SLUG
-          -e TRAVIS_TAG
-          -e NSVER
-          -e ENDPOINTS_PACK
-          "
-  script: |
-        set -e
-        docker run -ti \
-          --name makerpms ${EVARS} \
-          --hostname "b${TRAVIS_BUILD_NUMBER}.nethserver.org" \
-          --volume $PWD:/srv/makerpms/src:ro ${DOCKER_IMAGE} \
-          makerpms-travis *.spec
-        docker commit makerpms nethserver/build
-        docker run -ti ${EVARS} \
-          -e SECRET \
-          -e SECRET_URL \
-          -e AUTOBUILD_SECRET \
-          -e AUTOBUILD_SECRET_URL \
-          nethserver/build uploadrpms-travis
+This is an example of ``.github/workflows/make-rpms.yml`` contents: ::
+
+  name: Make RPMs
+  on:
+    push:
+      branches:
+        - master
+        - main
+    pull_request:
+  jobs:
+    make-rpm:
+      runs-on: ubuntu-latest
+      env:
+        dest_id: core
+        nsver: 7
+        docker_image: nethserver/makerpms:7
+      steps:
+        - uses: actions/checkout@v3
+          with:
+            fetch-depth: 0
+            ref: ${{ github.head_ref }}
+        - name: Generate .env file
+          run: |
+            cat > .env <<EOF
+              DEST_ID=${{ env.dest_id }}
+              NSVER=${{ env.nsver }}
+              DOCKER_IMAGE=${{ env.docker_image }}
+              GITHUB_ACTIONS=1
+              GITHUB_HEAD_REF=${{ github.head_ref }}
+              GITHUB_REF=${{ github.ref }}
+              GITHUB_REPOSITORY=${{ github.repository }}
+              GITHUB_RUN_ID=${{ github.run_id }}
+              ENDPOINTS_PACK=${{ secrets.endpoints_pack }}
+              SECRET=${{ secrets.secret }}
+              SECRET_URL=${{ secrets.secret_url }}
+              AUTOBUILD_SECRET=${{ secrets.autobuild_secret }}
+              AUTOBUILD_SECRET_URL=${{ secrets.autobuild_secret_url }}
+            EOF
+        - name: Run prep-sources if present.
+          run: if test -f "prep-sources"; then ./prep-sources; fi
+        - name: Build RPM and publish
+          run: |
+            echo "Starting build..."
+            docker run --name makerpms \
+              --env-file .env \
+              --hostname $GITHUB_RUN_ID-$GITHUB_RUN_NUMBER.nethserver.org \
+              --volume $PWD:/srv/makerpms/src:ro \
+               ${{ env.docker_image }} \
+              makerpms-github -s *.spec
+            echo "Build succesful."
+            if [[ "${{ secrets.endpoints_pack }}" && "${{ secrets.secret }}" ]]; then
+              echo "Publish configuration exists, pushing package to repo..."
+              docker commit makerpms nethserver/build
+              docker run \
+                --env-file .env \
+                nethserver/build \
+                uploadrpms-github
+              echo "Publish complete."
+            fi
+            rm .env
 
 Usage
 ^^^^^
 
 Travis CI builds are triggered automatically when:
```

Comment:
It seems there are still some `travis` to search and replace in this file, like this one.